### ISSUE TITLE
[Bug] "array.push something" becomes "array.pushsomething"

### ISSUE THREAD

**Issue by Glavin001** [sterpe/coffee-fmt#14]
I was formatting https://github.com/Glavin001/atom-beautify/blob/master/src/beautify.coffee#L31-L34 and it broke the code.

Before:

```
 posArray.push [
      bufferPosition.row
      bufferPosition.column
    ]
```

After:

```
 posArray.push[
      bufferPosition.row
      bufferPosition.column
    ]
```

**Comment by sterpe** [comment]
ah okay.  Let me see how I need to approach this.

**Comment by sterpe** [comment]
I was always hoping to get away without writing my own parser for coffeescriptvbut I think it's the only way to handle this and #9

**Comment by sterpe** [comment]
Fixed 3f8e5504b4eef76e5288d1addf9bb032f7afa38d
